sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -39,10 +48,11 @@ public DiscountDto getById(Long id) {
         return discountDto;
     }
 
+    //fixme What should this method do?
```

Comment:
If we already don't use it on UI we can remove this one. We can return it at any time from Git history if it will be required.